test/integration/features/provision_various_versions.feature

```diff
@@ -18,6 +18,12 @@ Feature: Provision all major OpenShift versions
       """
       <ocVersion>
       """
+      And "status code" of HTTP request to "/healthz" of OpenShift instance is equal to "200"
+      And "body" of HTTP request to "/healthz" of OpenShift instance contains "ok"
+      And "status code" of HTTP request to "/healthz/ready" of OpenShift instance is equal to "200"
+      And "body" of HTTP request to "/healthz/ready" of OpenShift instance contains "ok"
+      And "status code" of HTTP request to "/console" of OpenShift instance is equal to "200"
+      And "body" of HTTP request to "/console" of OpenShift instance contains "<title>OpenShift Web Console</title>"
```

Comment:
@amitkrout , are you deleting `v3.6.0` as it seems redundant to me?

Reply:
@budhrg Why do you think it is redundant, is it becaue we are already checking two versions less than v3.6.1, i.e. v1.4.1 and v1.5.1?

Reply:
Because, at least I assume v3.6.1 is patch release to v3.6.0

Reply:
Also, why we are not testing `1.5.0` it is also listed in `minishift openshift version list`

Reply:
> Because, at least I assume v3.6.1 is patch release to v3.6.0

Make sense

Reply:
Done